### DISCUSSION TITLE
can i remove meta viewport element?

### DISCUSSION THREAD

**Discussion by narrowizard** [vercel/next.js#17277] [Help] [answered]
i find that nextjs will auto add a meta element `<meta name="viewport" content="width=device-width">`.  
are there any way to remove this meta element?  

## context
As my project is not compatible with mobile view, i dont need the viewport meta element. then users can visit my site by zooming in at least.

**Comment by hassanmian** [comment]
Any update regarding this?

**Comment by jamesmosier** [comment, marked as answer]
There is no way to remove it completely, but you can use `next/head` in `pages/_app.js` to set a global viewport that overrides the default one.

```js
// pages/_app.js
import Head from 'next/head'

function MyApp({ Component, pageProps }) {
  return (
    <>
      <Head>
        <meta name="viewport" content="viewport-fit=cover" />
      </Head>
      <Component {...pageProps} />
    </>
  )
}

export default MyApp
```

(source taken from [here](https://github.com/vercel/next.js/blob/master/errors/no-document-viewport-meta.md))

**Reply by lucasvazq** [reply]
Hi @jamesmosier , this generates to me a duplicated viewport. One of that viewport is mine, the other is the default viewport.

Do you know how to fix it?
The same thing happens to me with the charset tag.

**Reply by lucasvazq** [reply]
I found the solution

https://github.com/vercel/next.js/blob/master/errors/no-document-viewport-meta.md
